access_log/access_log_record.go

```diff
@@ -66,9 +68,16 @@ func (r *AccessLogRecord) makeRecord() *bytes.Buffer {
 	if r.RouteEndpoint == nil {
 		fmt.Fprintf(b, "app_id:MissingRouteEndpointApplicationId")
 	} else {
-		fmt.Fprintf(b, `app_id:%s`, r.RouteEndpoint.ApplicationId)
+		fmt.Fprintf(b, `app_id:%s `, r.RouteEndpoint.ApplicationId)
 	}
 
+	extraHeadersToLog := make(map[string]string)
+	for _, header := range config.ExtraHeadersToLog {
+		extraHeadersToLog[header] = r.FormatRequestHeader(header)
+	}
+	marshalledExtraHeadersToLog, _ := json.Marshal(extraHeadersToLog)
```

Comment:
hey @simonjohansson what is the cost of this logging  ? 
What about a test if config.ExtraHeadersToLo is empty there is no need to make the all process like :
```go
if config.ExtraHeadersToLog != nil {
extraHeadersToLog := make(map[string]string)
	for _, header := range config.ExtraHeadersToLog {
		extraHeadersToLog[header] = r.FormatRequestHeader(header)
	}
marshalledExtraHeadersToLog, _ := json.Marshal(extraHeadersToLog)
fmt.Fprintf(b, `headers:%s`, string(marshalledExtraHeadersToLog))
}
```